modules/monitoring-querying-metrics.adoc

```diff
@@ -0,0 +1,14 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/managing-metrics.adoc
+
+[id="querying-metrics_{context}"]
+= Querying metrics
+
+[role="_abstract"]
+The {product-title} monitoring dashboard enables you to run Prometheus Query Language (PromQL) queries to examine metrics visualized on a plot. This functionality provides information about the state of a cluster and any user-defined workloads that you are monitoring.
+
+As a *cluster administrator*, you can query metrics for all core {product-title} and user-defined projects.
```

Comment:
I don't think is applicable for OSD

Reply:
@fahlmant - Do I remove both paragraphs, or just "As a *cluster administrator*, you can query metrics for all core {product-title} and user-defined projects."

Reply:
Dedicated-admins can only query across one namespace at a time via the OpenShift console directly, so this sentence, plus any references to "all core projects" or "state of a cluster"

Reply:
Ok made the changes.